HoneyBadgers.WebApp/Startup.cs

```diff
@@ -23,13 +32,34 @@ public Startup(IConfiguration configuration)
         public void ConfigureServices(IServiceCollection services)
         {
             services.AddControllersWithViews().AddRazorRuntimeCompilation();
-            services.AddTransient<IMovieRepository, MovieRepository>();
-            services.AddTransient<IUserRepository, UserRepository>();
+            services.AddSession();
+            services.AddHttpContextAccessor();
+            services.AddAuthentication(CookieAuthenticationDefaults.AuthenticationScheme).AddCookie();
+            services.TryAddSingleton<IHttpContextAccessor, HttpContextAccessor>();
+            services.AddTransient<AuthService>();
+            services.AddTransient<UserService>();
+            services.AddTransient<IFavoriteMoviesService, FavoriteMoviesService>();
```

Comment:
Co daje wywołanie <IService,Service>  ?

Reply:
myślę że poprostu precyzyjne zdefiniowanie który serwis, czyli ten serwis, który dziedziczy po tym interfejsie. To jest coś o co sama chcę zapytać, bo dla swoich serwisów nie robiłam interfejsów, bo miałam poczucie że to jest przerost formy nad treścią, skoro podany interfejs ma maksymalnie jeden serwis